lib/core/bufferFactory.cpp

```diff
@@ -1,10 +1,10 @@
 #include "bufferFactory.hpp"
 
 #include "Config.hpp"         // for Config
+#include "VisFrameView.hpp"   // for VisFrameView
```

Comment:
Interesting that one of our auto formatters is case sensitive in it's sorting algorithm.  @nritsche any idea if that's easy to change?

Reply:
> I'm not sure that it makes sense to rename `visBuffer` to `VisFrameView`. It seems to me in most places what we are really referring to is the `VisBuffer` not the frame view, the frame view is just constructed on top for convenience. @jrs65 any thoughts on this?

I don't mind either way. It's always been a bit odd to call it that, but there's no actual class called visBuffer.

However, for the moment I'd probably just leave it called visBuffer.*. It sounds like we're probably going to have a big reorg and rename at somepoint soon, so we might as well just do it in that.

Reply:
> Interesting that one of our auto formatters is case sensitive in its sorting algorithm. @nritsche any idea if that's easy to change?

There's no option in clang-format to change that behavior.